### ISSUE TITLE
Bug: Unlimited Storagetime and Download buggy

### ISSUE THREAD

**Issue by Jisagi** [Forceu/Gokapi#67]
## Issue
When uploading a file as descibed in the api docs you do get a file which is listed as Unlimited for both the Downloads remaining as well as Stored until time. The issue is, the hotlink for image files does not work and only shows the "The requested image has expired" text.

## Steps to reproduce
Upload an image via the normal frontend with unlimited storetime and downloads.
The payload will be like this. The 1 and 30 are just the last values I used, but the checkboxes are NOT checked for both, so they both get ignored.
```
allowedDownloads: 1
expiryDays: 30
password: 
isUnlimitedDownload: true
isUnlimitedTime: true
file: (binary)
```

Uploading a file as told by the docs omits `isUnlimitedDownload` and `isUnlimitedTime`. This causes the hotlink to not work for images but everything else to work just fine. A payload therefore looks like this.
```
allowedDownloads: 0
expiryDays: 0
password: 
file: (binary)
```

I think using the 0 for both values and omitting the other two payload contents causes this to happen.

EDIT:
~This only happens when both values are supposed to be unlimited. As soon as one has a set limit, this does not happen anymore.~

EDIT2:
This happens when both values are unlimited and also when the downloads remaining aren't unlimited but the stored until is.

**Comment by Forceu** [comment]
Thanks, I will have a look at it tomorrow!

**Comment by Forceu** [comment]
Fixed in 803016af3c89dcfe9bb1045e1ea937c2e855493d

**Comment by Jisagi** [comment]
Works like a charm, thank you!
